website/docs/r/mobile_network_sim_group.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "Mobile Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_mobile_network_sim_group"
+description: |-
+  Manages a Mobile Network Sim Group.
+---
+
+# azurerm_mobile_network_sim_group
+
+Manages a Mobile Network Sim Group.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_mobile_network" "example" {
+  name                = "example-mn"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  mobile_country_code = "001"
+  mobile_network_code = "01"
+}
+
+data "azurerm_user_assigned_identity" "example" {
+  name = "example-identity"
+}
+
+data "azurerm_key_vault_key" "example" {
+  name         = "some-key"
+  key_vault_id = data.azurerm_key_vault.example.id
+}
+
+resource "azurerm_mobile_network_sim_group" "example" {
+  name                = "example-mnsg"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  mobile_network_id   = azurerm_mobile_network.example.id
+  encryption_key_url  = data.azurerm_key_vault_key.example.id
+
+  identity {
+    type         = "SystemAssigned, UserAssigned"
+    identity_ids = [data.azurerm_user_assigned_identity.example.id]
+  }
+
+  tags = {
+    key = "value"
+  }
+
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name which should be used for this Mobile Network Sim Groups. Changing this forces a new Mobile Network Sim Group to be created.
+
+* `resource_group_name` - (Required) Specifies the name of the Resource Group where the Mobile Network Sim Groups should exist. Changing this forces a new Mobile Network Sim Group to be created.
```

Comment:
can this resource belong to a different resource group then the one in mobile_network_id ?

Reply:
oh, it must be in the same resource group with mobile network. removed it and going to parse from mobile network id.